pyproject.toml

```diff
@@ -9,18 +9,18 @@ packages = [{include = "iwf"}]
 [tool.poetry.dependencies]
 python = "^3.9"
 python-dateutil = { version = "^2.8.2", python = "<3.11" }
+httpx = "0.28.1"
 
 [tool.poetry.dev-dependencies]
-pytest = "^7.2.1"
-mypy = "^1.1.1"
-isort = "^5.11.4"
-pre-commit = "^3.0.1"
-wemake-python-styleguide = "^0.17.0"
-pytest-cov = "^4.0.0"
-anyio = "^3.6.2"
-pytest-env = "^0.8.1"
-httpx = "^0.23.3"
-openapi-python-client = "^0.14.0"
+pytest = "^8.3.5"
+mypy = "^1.15.0"
+isort = "^6.0.1"
+pre-commit = "^4.2.0"
+wemake-python-styleguide = "^0.19.2"
+pytest-cov = "^6.1.1"
+anyio = "^4.9.0"
+pytest-env = "^1.1.5"
+openapi-python-client = "^0.24.3"
```

Comment:
All of the dev dependencies have also been updated to the latest available version